src/core/p5.Graphics.js

```diff
@@ -76,28 +76,23 @@ p5.Graphics.prototype = Object.create(p5.Element.prototype);
  * function setup() {
  *   createCanvas(100, 100);
  *   stroke(255);
- *   colorMode(HSB);
+ *   fill(0);
  *
  *   // create and draw the background image
  *   bg = createGraphics(100, 100);
- *   bg.background(100);
- *   bg.colorMode(HSB);
- *   bg.strokeWeight(3);
- *   for (var i = 0; i < 100; i++) {
- *     bg.stroke(random(0, 255), 255, 255);
- *     bg.line(random(0, 100), random(0, 100), random(0, 100), random(0, 100));
- *   }
+ *   bg.ellipse(50, 50, 80, 80);
  * }
  * function draw() {
  *   var t = millis() / 1000;
  *   // draw the background
+ *   background(200);
  *   if (bg) {
  *     image(bg, frameCount % 100, 0);
  *     image(bg, frameCount % 100 - 100, 0);
  *   }
  *   // draw the foreground
- *   fill((t * 10) % 255, 255, 255);
- *   ellipse(50 + 50 * sin(t * 1.1713), 50 + 50 * cos(t * 1.5415), 30, 30);
+ *   var p = p5.Vector.fromAngle(t, 35).add(50, 50);
+ *   ellipse(p.x, p.y, 30);
  * }
```

Comment:
I'm still wondering about lines 88-95. Is there a way to write this without modulo and without using a p5.Vector? Does the bg object need to be drawn twice?

Reply:
yeah, for the scrolling background, using the modulo (to get the repitition) and the duplicate drawing (to get both left & right sides) is the simplest way to go. the foreground is an ellipse going around in a circle, i'm not sure how to do this without using trig or fromAngle.

i wanted to give an example of code that needs to use a `Graphics` object (eg, re-rendering a cached canvas) for context - rather than just a trivial two-liner that doesn't really say anything more than the text.

Reply:
ok I see. maybe we do two examples then? one that shows the most minimal use of the function just to clearly illustrate the syntax usage, even if it doesn't really warrant a graphics buffer? then this one, with a little more commenting to explain what's going on, either inline or at the top?